variants/esp32-eth-clk-out0-spiram/README.md

```diff
@@ -0,0 +1,11 @@
+# ESP32-ETH-CLK-OUT17-SPIRAM
```

Comment:
Why is this in caps here? Maybe that is what we do in the other individual READMEs.

Reply:
That's what we did so far.
If we want to change it, we should change it for all READMEs.